apps/meteor/ee/client/voip/modal/DialPad/hooks/useDialPad.tsx

```diff
@@ -41,18 +47,33 @@ export const useDialPad = ({ initialValue, errorMessage }: DialPadProps): DialPa
 	});
 
 	const { ref, onChange } = register('PhoneInput');
+
 	const value = watch('PhoneInput');
 
 	const [disabled, setDisabled] = useState(true);
 
 	const handleBackspaceClick = useCallback((): void => {
+		clearErrors();
 		setValue('PhoneInput', value.slice(0, -1));
-	}, [setValue, value]);
+	}, [clearErrors, setValue, value]);
 
 	const handlePadButtonClick = useCallback(
 		(digit: string | number): void => {
+			clearErrors();
+
 			setValue('PhoneInput', value + digit);
 		},
+		[clearErrors, setValue, value],
+	);
+
+	const handlePadButtonLongPressed = useCallback(
+		(digit: string | number): void => {
+			if (digit === 0) {
+				return setValue('PhoneInput', `${value}+`);
+			}
```

Comment:
From what I see the `digit` should already be + in this scenario. Isn't this redundant?

Reply:
<img width="469" alt="image" src="https://user-images.githubusercontent.com/17487063/178804337-e4f28bb3-8a1e-4593-a40e-5dfbf0bd7d38.png">
Here is digits... there is only 0